.github/workflows/ci_standalone.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: Galaxy CI
+on: {pull_request: {branches: ['*']}, push: {branches: ['*']}}
+jobs:
+
+  standalone_integration:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+
+      - name: Install docker-compose
+        run: pip3 install --upgrade docker-compose
+
+      - name: clone the hub ui repo
+        run: git clone https://github.com/ansible/ansible-hub-ui /tmp/ansible-hub-ui
+
+      - name: create the .compose.env file
+        run: rm -f .compose.env; cp .compose.env.example .compose.env
+
+      - name: set the hub ui path in .compose.env
+        run: echo "ANSIBLE_HUB_UI_PATH='/tmp/ansible-hub-ui'" >> .compose.env
+
+      - name: disable approval setting override
+        run: sed -i.bak 's/PULP_GALAXY_AUTO_SIGN_COLLECTIONS/#PULP_GALAXY_AUTO_SIGN_COLLECTIONS/' dev/standalone/galaxy_ng.env
```

Comment:
I guess you meant AUTO_APPROVAL instead of AUTO_SiGN

Reply:
@rochacbruno ah yes, good catch, ty